components/topbar/user/src/index.js

```diff
@@ -225,19 +223,22 @@ export default function TopbarUser({
           </div>
         </div>
       </div>
-      <div className="sui-TopbarUser-ctaButton">
-        <AtomButton
-          link
-          href={navCtaUrl}
-          title={navCtaText}
-          leftIcon={<navCTA.icon svgClass="sui-TopbarUser-ctaButtonIcon" />}
-          size="small"
-          type="primary"
-          onClick={onCTAClick}
-        >
-          {navCtaText}
-        </AtomButton>
-      </div>
+      {customContent}
+      {navCTA && !customContent && (
+        <div className="sui-TopbarUser-ctaButton">
+          <AtomButton
+            link
+            href={navCTA.url}
+            title={navCTA.text}
+            leftIcon={<navCTA.icon svgClass="sui-TopbarUser-ctaButtonIcon" />}
+            size="small"
+            type="primary"
```

Comment:
🍆 I know it's unrelated to this `PR` but perhaps you could use size and type values as imported from `AtomButton`?

Reply:
I can, yes